invokeai/frontend/web/src/app/store/middleware/listenerMiddleware/listeners/enqueueRequestedCanvas.ts

```diff
@@ -123,16 +123,14 @@ export const addEnqueueRequestedCanvasListener = (startAppListening: AppStartLis
         const batchId = enqueueResult.batch.batch_id as string; // we know the is a string, backend provides it
 
         // Prep the canvas staging area if it is not yet initialized
-        if (!state.canvas.layerState.stagingArea.boundingBox) {
-          dispatch(
-            stagingAreaInitialized({
-              boundingBox: {
-                ...state.canvas.boundingBoxCoordinates,
-                ...state.canvas.boundingBoxDimensions,
-              },
-            })
-          );
-        }
+        dispatch(
+          stagingAreaInitialized({
+            boundingBox: {
+              ...state.canvas.boundingBoxCoordinates,
+              ...state.canvas.boundingBoxDimensions,
+            },
+          })
+        );
```

Comment:
Uses can continually enqueue with an active staging area, but this will delete their currently-staged images.

Reply:
So then it sounds like clearing the staging area on cancel would also not work in theory?

Reply:
Correct, won't work. I've just updated the PR with a solution, will comment in main thread